### PR TITLE
Feat: [#1109] - extended compliance finding - compliance by adding co…

### PR DESCRIPTION
**Pull request by PavelJurka**
https://github.com/ocsf/ocsf-schema/issues/1109

The extension is:
compliance finding . compliance with:

compliance_references: Array of KB Article
compliance_standards: Array of KB Article

### CODE REVIEW

**Review by irakledibm — DISMISSED** [review]
lgtm

**Review by jasonbreimer — DISMISSED** [review]
Looks good.

**Review by davemcatcisco — APPROVED** [review]

**Review by davemcatcisco — APPROVED** [review]

**Review by irakledibm — APPROVED** [review]

File: objects/compliance.json [hunk 1]
```diff
@@ -4,6 +4,12 @@
   "extends": "object",
   "name": "compliance",
   "attributes": {
+    "compliance_references": {
+      "requirement": "optional"
+    },
+    "compliance_standards": {
```
Comment (floydtree) [inline objects/compliance.json:10]: A quick question, how do you want to distinguish between this new attribute and the existing `standards` attribute?

File: objects/compliance.json [hunk 2]
```diff
@@ -4,6 +4,12 @@
   "extends": "object",
   "name": "compliance",
   "attributes": {
+    "compliance_references": {
```
Comment (floydtree) [inline objects/compliance.json:7]: Are you specifically looking for a kb_article type object for this one? If not, there's a simple `references` attribute available in the schema. A few samples of the target data will be helpful.

**Review by floydtree — CHANGES_REQUESTED** [review]

File: objects/compliance.json (hunk 1 above)
Comment (PavelJurka) [reply objects/compliance.json:10]: The standards field is only an array of string - we would like to have i more sophisticated - title, name of a standard, url link, etc. and KB Article works perfectly.

File: objects/compliance.json (hunk 2 above)
Comment (PavelJurka) [reply objects/compliance.json:7]: The same as previous - references is only a plain string array - we'd like to have it more sophisticated -> - title, name of a standard, url link, etc. and KB Article works perfectly.

**Review by floydtree — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -58,6 +58,8 @@ Thankyou! -->
     7. Added `whois` object. #992
     8. Added `domain_contact` and array-typed `domain_contacts` object for use with `whois` object. #992
     9. Added `Windows Service` object to the Windows extension. #1103
+    10. Added array-typed `compliacne_references` and array-typed `compliance_standards` objects as array of `kb_article` and used in `compliance` object. #1110
```
Comment (Aniak5) [inline CHANGELOG.md:61]: This is misspelled and should be `compliance_references`.

**Pull request merged**